zerver/lib/markdown/api_return_values_table_generator.py

```diff
@@ -160,11 +160,21 @@ def render_table(self, return_values: Dict[str, Any], spacing: int) -> List[str]
                 elif return_values[return_value]["additionalProperties"].get(
                     "additionalProperties", False
                 ):
```

Comment:
The pattern for the recursive call seems to be:
1. data type
2. description
3. properties

It's pretty safe to assume that if these description calls are happening, then the data type is an `object`, but to be sure maybe add in the `generate_data_type` call here too?